### ISSUE TITLE
Commends are not getting counted!

### ISSUE THREAD

**Issue by rezac930** [ValveSoftware/Dota2-Gameplay#12745]
### Description

I have been playing Dota 2 for several months.
The number of my commends is stuck at 48!
But when the game is on versus screen, it is written on my info that I have received 140 commends in the last 15 games.
I saw many people on the internet who reported this problem but it was not addressed.
Please fix this problem.

**Comment by ligaapl** [comment]
Yes. I have 2172 commends a 2 month ago and commends still not counted after profile update! ID 113184848

**Comment by Aluerie** [comment]
#11278
